---a/net/minecraft/server/gui/PlayerListComponent.java
+++b/net/minecraft/server/gui/PlayerListComponent.java
@@ -1,37 +1,35 @@
 package net.minecraft.server.gui;
 
 import java.util.Vector;
-import javax.swing.JList;
+
+import javax.swing.*;
+
 import net.minecraft.entity.player.EntityPlayerMP;
 import net.minecraft.server.MinecraftServer;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-@SideOnly(Side.SERVER)
-public class PlayerListComponent extends JList implements IUpdatePlayerListBox
-{
+
+public class PlayerListComponent extends JList implements IUpdatePlayerListBox {
+
     private MinecraftServer server;
     private int ticks;
     private static final String __OBFID = "CL_00001795";
 
-    public PlayerListComponent(MinecraftServer server)
-    {
+    public PlayerListComponent(MinecraftServer server) {
         this.server = server;
         server.registerTickable(this);
     }
 
-    public void update()
-    {
-        if (this.ticks++ % 20 == 0)
-        {
+    public void update() {
+        if (this.ticks++ % 20 == 0) {
             Vector vector = new Vector();
 
-            for (int i = 0; i < this.server.getConfigurationManager().playerEntityList.size(); ++i)
-            {
-                vector.add(((EntityPlayerMP)this.server.getConfigurationManager().playerEntityList.get(i)).getName());
+            for (int i = 0; i < this.server.getConfigurationManager().playerEntityList.size(); ++i) {
+                vector.add(((EntityPlayerMP) this.server.getConfigurationManager().playerEntityList.get(i)).getName());
             }
 
             this.setListData(vector);
         }
     }
-}
+}
